tests/conftest.py

```diff
@@ -10,9 +10,122 @@
+        self.contract = contract
+
+    def set_account(
+        self,
+        a_address,
+        b_address,
+        *,
+        creditline_given=0,
+        creditline_received=0,
+        interest_rate_given=0,
+        interest_rate_received=0,
+        is_frozen=False,
+        m_time=0,
+        balance=0,
+    ):
+        self.contract.functions.setAccount(
+            a_address,
+            b_address,
+            creditline_given,
+            creditline_received,
+            interest_rate_given,
+            interest_rate_received,
+            is_frozen,
+            m_time,
+            balance,
+        ).transact()
+
+    def check_account(
+        self,
+        a_address,
+        b_address,
+        *,
+        creditline_given=None,
+        creditline_received=None,
+        interest_rate_given=None,
+        interest_rate_received=None,
+        is_frozen=None,
+        m_time=None,
+        balance=None,
+    ):
+        account_tuple = self.contract.functions.getAccount(a_address, b_address).call()
+
+        for value, index in enumerate(
+            [
+                creditline_given,
+                creditline_received,
+                interest_rate_given,
+                interest_rate_received,
+                is_frozen,
+                m_time,
+                balance,
+            ]
+        ):
+            if value is not None:
+                if value != account_tuple[index]:
+                    return False
+            return True
+
+    def balance(self, a_address, b_address):
+        return self.contract.functions.balance(a_address, b_address).call()
+
+    def creditline(self, creditor_address, debtor_address):
+        return self.contract.functions.creditline(
+            creditor_address, debtor_address
+        ).call()
+
+    def interest_rate(self, creditor_address, debtor_address):
+        return self.contract.functions.interestRate(
+            creditor_address, debtor_address
+        ).call()
+
+    def update_trustline(
+        self,
+        creditor_address,
+        debtor_address,
+        *,
+        creditline_given=0,
+        creditline_received=0,
+        interest_rate_given=0,
+        interest_rate_received=0,
+        is_frozen=False,
+        accept=False,
+    ):
+        self.contract.functions.updateTrustline(
+            debtor_address,
+            creditline_given,
+            creditline_received,
+            interest_rate_given,
+            interest_rate_received,
+            is_frozen,
+        ).transact({"from": creditor_address})
+
+        if accept:
+            self.contract.functions.updateTrustline(
+                creditor_address,
+                creditline_received,
+                creditline_given,
+                interest_rate_received,
+                interest_rate_given,
+                is_frozen,
+            ).transact({"from": debtor_address})
+
+    def transfer(self, value: int, *, path, max_fee=0, extra_data=EXTRA_DATA):
```

Comment:
I guess a default of max_uint64 would be better for max_fee.

Reply:
makes sense

Reply:
Makes sense